### PR TITLE
feat(owners): Owner Rule Builder

### PR DESCRIPTION
**Pull request by MaxBittker**
localhost_8000_settings_organization_sentry_project_ludic-science_ownership_ 3
localhost_8000_settings_organization_sentry_project_ludic-science_ownership_ 2
image

approximating:
image

### CODE REVIEW

**Review by billyvg — COMMENTED** [review]
This needs some polish
image

I think we can just remove the headers and just have a border between teams + members?

File: src/sentry/static/sentry/app/views/settings/project/projectOwnership/ruleBuilder.jsx [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+            <BuilderDropdownButton isOpen={isOpen} size="xsmall">
+              {t('Add Owners')}
+            </BuilderDropdownButton>
+          )}
+        </BuilderDropdownAutoComplete>
+        <RuleAddButton
+          priority="primary"
+          onClick={this.handleAddRule.bind(this)}
+          icon="icon-circle-add"
+          style={{width: '50px'}}
```
[the remark this answers is not in the document]
Comment (billyvg) [reply src/sentry/static/sentry/app/views/settings/project/projectOwnership/ruleBuilder.jsx]: I think having it 32x32 to match the avatars would be nice

File: src/sentry/static/sentry/app/views/settings/project/projectOwnership/ruleBuilder.jsx [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+    return (
+      <BuilderBar>
+        <BuilderSelect value={type} onChange={this.handleTypeChange.bind(this)}>
+          <option value="path">Path</option>
+          <option value="url">URL</option>
+        </BuilderSelect>
+        <BuilderInput
+          controlled
+          value={text}
+          onChange={this.onValueChange.bind(this)}
```
[the remark this answers is not in the document]
Comment (MaxBittker) [reply src/sentry/static/sentry/app/views/settings/project/projectOwnership/ruleBuilder.jsx]: I don't follow

[the remark this answers is not in the document]
Comment (billyvg) [reply src/sentry/static/sentry/app/views/settings/project/projectOwnership/ruleBuilder.jsx]: `onChangeValue`

**Review by billyvg — APPROVED** [review]
🛳

**Pull request merged**